lib/version.rb

```diff
@@ -3,4 +3,4 @@
 # Copyright 2013 whiteleaf. All rights reserved.
 #
 
-Version = "3.1.7"
+Version = "3.1.8"
```

Comment:
__[RuboCop]__

Style/ConstantName: Use SCREAMING_SNAKE_CASE for constants.


<!-- Comment by [SideCI](https://sideci.com) -->